packages/@aws-cdk/aws-ecs-patterns/lib/fargate/network-load-balanced-fargate-service.ts

```diff
@@ -135,5 +140,10 @@ export class NetworkLoadBalancedFargateService extends NetworkLoadBalancedServic
       cloudMapOptions: props.cloudMapOptions,
     });
     this.addServiceAsTarget(this.service);
+    // All public-facing NLB fronted fargate service with assignPublicIp by defaults allow from any IPv4 traffic
+    const publicNLB = props.publicLoadBalancer !== undefined ? props.publicLoadBalancer : true;
+    if (publicNLB && props.assignPublicIp) {
+      this.service.connections.connections.allowFromAnyIpv4(Port.tcp(containerPort));
```

Comment:
This is not necessarily true, and it would be a breaking change. I think we should rely on users to determine if they want to allow the port or not.

Reply:
@pkandasamy91 since `containerPort` is being set in [line 128](https://github.com/aws/aws-cdk/pull/4856/files#diff-34db4be841bead71ec835c013f79f8a9R128), this effectively is allowing the customer to determine which port the NLB is receiving traffic on, no? Would this still be a breaking change?